src/ophys_etl/filters/filters.py

```diff
@@ -0,0 +1,16 @@
+from typing import List
+
+from scipy.sparse import coo_matrix
+
+
+def filter_rois_by_longest_edge_length(coo_rois: List[coo_matrix],
+                                       longest_edge_threshold: int) -> \
+        List[coo_matrix]:
+    filtered_rois = []
```

Comment:
Docstring?

Reply:
Added docstring and explicitly stated less than not equal to in the docstring